src/platform/unix.rs

```diff
@@ -320,8 +320,8 @@ pub trait WindowBuilderExtUnix {
     fn with_class(self, class: String, instance: String) -> WindowBuilder;
     /// Build window with override-redirect flag; defaults to false. Only relevant on X11.
     fn with_override_redirect(self, override_redirect: bool) -> WindowBuilder;
-    /// Build window with `_NET_WM_WINDOW_TYPE` hint; defaults to `Normal`. Only relevant on X11.
-    fn with_x11_window_type(self, x11_window_type: XWindowType) -> WindowBuilder;
+    /// Build window with `_NET_WM_WINDOW_TYPE` hints; defaults to `Normal`. Only relevant on X11.
+    fn with_x11_window_type(self, x11_window_type: &'static [x11::util::WindowType]) -> WindowBuilder;
```

Comment:
`&'static`? Adding a lifetime to the `WindowBuilder` I think is better then `&'static`. glutin does something similar with `ContextBuilder`, so it wouldn't be out of place.

Reply:
You're right, it's less of a deal than I thought it was.